flags.go

```diff
@@ -76,6 +77,11 @@ func defineFlagVar() *FlagVar { //nolint:funlen
 		"gateway-selector", "Label selector of the gateway resource that the virtual service will use. "+
 			"Ignored if gateway-ns-name flag is specified. Format: K8s label selector expression. "+
 			"Example: \"label1=value1,label2=value2\"")
+	flag.BoolVar(&flagVar.enableWatcherLocalTesting, "enable-watcher-local-testing", false,
+		"Enabling KCP Watcher two-cluster setup to be tested locally using k3d")
+	flag.IntVar(&flagVar.listenerHTTPPortLocalMapping, "listener-http-local-mapping",
+		defaultListenerHTTPPortLocalMapping, `Port that is mapped to HTTP port of the local k3d cluster using
```

Comment:
Why does this need to be set? What came of the solution with the NodePort? Did that not work?

Reply:
Actually, I haven't tested if Node-port works as we already decided not to go with Node-port after my first investigation.

Reply:
Very sorry that I didnt have this in mind. Could you quickly summarise again why we said we didnt want to go for nodeport? I lost that memory it seems and I am not really sure in which ticket we tracked that decision anymore X(

Reply:
I think the main reason was that we agreed to support only k3d for local testing and the port mapping was the recommended way to do it.